### ISSUE TITLE
Upload Windows PDBs to Backtrace (add /pdbpagesize:8192 support)

### ISSUE THREAD

**Issue by goodov** [brave/brave-browser#26890]
Currently we upload `.sym` files to Backtrace on all platforms, but this is wrong on Windows, because Backtrace can use PDBs directly to create more precise stack traces.

Backtrace recommends uploading `.exe` and `.dll` files to improve stack unwinding, because some information is available only in these binaries.

Backtrace supports `/pdbpagesize:8192`, which means we can use `symbol_level = 2` to keep all debug information. Currently we had to set `symbol_level = 1` to make our Release builds work on Windows.

Note: we currently upload `symbols` archive to GitHub with `.sym` files, which now doesn't make sense, because we added `native-symbols` archive with PDBs which is actually needed to debug dumps, so it's safe to just stop uploading `symbols` archive.

TL;DR:
1. remove `dump_syms` step on Windows.
2. upload `.exe, .dll, .pdb` to Backtrace.
3. stop uploading `symbols` archive to GitHub in Windows builds.
4. set `/pdbpagesize:8192` by default.

Backtrace support answer on `/pdbpagesize:8192`:
![image](https://user-images.githubusercontent.com/5928869/202988458-3ddffd3a-af84-4995-98eb-e08056fc5118.png)

**Comment by goodov** [comment]
Chromium is on the way to enable large PDBs, it should be around the same time when they'll switch to VS 2022 (end of Jan 2022 is expected date).

https://bugs.chromium.org/p/chromium/issues/detail?id=1404175
https://chromium-review.googlesource.com/c/chromium/src/+/4128591

**Comment by goodov** [comment]
large PDBs are now enabled in Chromium, but there's still an option to disable it.
https://groups.google.com/a/chromium.org/g/chromium-dev/c/wk4-YGSGIic

cc @cdesouza-chromium @emerick if Chromium rebase will fail on `dump_syms` step, you need to set `use_large_pdbs=false` gn arg for now.

**Comment by goodov** [comment]
I have all preparations on our side ready, but testing revealed that Backtrace doesn't parse big PDBs properly for some reason. I asked their support about this. Will update later.

**Comment by goodov** [comment]
It turned out, Backtrace wasn't ready to handle new and shiny Brave binaries built with the latest LLVM toolchain. We spent around 4 months resolving all issues that was blocking us from using PDBs on Backtrace directly, but now everything is finally resolved and crash dumps parsed with PDBs and DLLs look amazingly well:

https://github.com/brave/brave-browser/assets/5928869/e58823f0-9b3c-4ac3-96d4-f08f770231f3

**Comment by goodov** [comment]
Backtrace uses PDBs for all windows builds now. We created symbol server that covers this process automatically.

related issues:
https://github.com/brave/devops/issues/9967
https://github.com/brave/devops/issues/10633
